lib/general.gi

```diff
@@ -196,18 +196,33 @@ BindGlobal( "WriteBrentFactorsFiles",
 ##
 #F  FetchBrentFactors( ) . . get Brent's tables of factors of numbers b^k - 1
```

Comment:
This comment needs to be updated too, to reflect 2 arguments.

@Stefan-Kohl is this function undocumented and not used anywhere in the package code? Otherwise, those places may need an update too.

Reply:
@alex-konovalov  The function FetchBrentFactors is undocumented, and not called anywhere from the package code. It is declared in lib/factint.gd, and the code is in lib/general.gi. These two places need updating.

Reply:
Ah, I see. I will update comments in both files then, and make a new release.

Reply:
@alex-konovalov Thank you!